BDArmory/CounterMeasure/VesselECMJInfo.cs

```diff
@@ -156,8 +160,45 @@ void UpdateJammerStrength()
             {
                 rcsr = 1;
             }
+			
+            TargetInfo ti = RadarUtils.GetVesselRadarSignature(vessel);
```

Comment:
Makes more sense to setup modified RCS from jamming once instead of every fixedUpdate tick in RadarUtils